ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx

```diff
@@ -281,6 +282,11 @@ export const WorkflowNodeInfo = (props: Props) => (
                         </div>
                     )
                 },
+                {
+                    title: 'EVENTS',
+                    key: 'events',
+                    content: <EventsPanel namespace={props.workflow.metadata.namespace} fieldSelector={'involvedObject.kind=Pod,involvedObject.name=' + props.node.name} />
```

Comment:
![image](https://user-images.githubusercontent.com/1142830/89853570-94458880-db46-11ea-9170-08962427e4eb.png)

we should probably include `kind in (Pod,PodDisruptionBudget,PersistentVolumeClaim`